Examples/KlaviyoSwiftExamples/Shared/CheckoutView.swift

```diff
@@ -0,0 +1,216 @@
+                            ScrollView {
+                                LazyVStack(spacing: 12) {
+                                    ForEach(Array(Set(appState.cartItems.map(\.id))), id: \.self) { itemId in
+                                        if let item = appState.cartItems.first(where: { $0.id == itemId }) {
+                                            CheckoutItemRow(item: item, appState: appState)
+                                                .padding(.horizontal, 16)
+                                        }
+                                    }
+                                }
+                                .padding(.vertical, 16)
+                            }
+
+                            // Order Total
+                            VStack(spacing: 16) {
+                                Divider()
+
+                                HStack {
+                                    Text("Order Total:")
+                                        .font(.title2)
+                                        .fontWeight(.medium)
+
+                                    Spacer()
+
+                                    Text("$\(String(format: "%.2f", totalPrice))")
+                                        .font(.title)
+                                        .fontWeight(.bold)
+                                        .foregroundColor(.red)
+                                }
+                                .padding(.horizontal, 20)
+
+                                // Action Buttons
+                                HStack(spacing: 16) {
+                                    Button("Back to Menu") {
+                                        dismiss()
+                                    }
+                                    .font(.headline)
+                                    .foregroundColor(.blue)
+                                    .frame(maxWidth: .infinity)
+                                    .padding(.vertical, 16)
+                                    .background(Color(.systemGray6))
+                                    .cornerRadius(12)
+
+                                    Button("Check Out") {
+                                        checkout()
+                                    }
+                                    .font(.headline)
+                                    .foregroundColor(.white)
+                                    .frame(maxWidth: .infinity)
+                                    .padding(.vertical, 16)
+                                    .background(
+                                        LinearGradient(
+                                            gradient: Gradient(colors: [.green, .green.opacity(0.8)]),
+                                            startPoint: .leading,
+                                            endPoint: .trailing
+                                        )
+                                    )
+                                    .cornerRadius(12)
+                                }
```

Comment:
```suggestion
                        HStack {
                            Button {
                                dismiss()
                            } label: {
                                Text("Back to Menu")
                                    .frame(maxWidth: .infinity)
                            }
                            .buttonStyle(.bordered)
                            .controlSize(.large)
                            .frame(maxWidth: .infinity)
                            
                            Button(action: checkout) {
                                Text("Check Out")
                                    .frame(maxWidth: .infinity)
                            }
                            .buttonStyle(.borderedProminent)
                            .tint(.green)
                            .controlSize(.large)
                        }
```

let's just use stock system button styles here